src/ring_theory/localization.lean

```diff
@@ -1153,6 +1153,132 @@ def at_one [is_localization.away (1 : R) S] : R ≃ₐ[R] S :=
 
 end at_units
 
+section localization_localization
+
+variable (M)
+
+variables (N : submonoid S) (T : Type*) [comm_ring T] [algebra S T]
+variables [algebra R T] [is_scalar_tower R S T]
+
+/--
+Given submodules `M ⊆ R` and `N ⊆ S = M⁻¹R`, with `f : R →+* S` the localization map, we have
+`N ⁻¹ S = T = (f⁻¹ (N • f(M))) ⁻¹ R`. I.e., the localization of a localization is a localization.
+-/
+lemma localization_localization_is_localization [is_localization N T] :
+  is_localization ((N ⊔ M.map (algebra_map R S)).comap (algebra_map R S).to_monoid_hom) T :=
+{ map_units := begin
+    rintros ⟨y, h⟩,
+    erw submonoid.mem_sup at h,
```

Comment:
Does it affect performance, usability downstream etc., when the goal is of type `Prop`?
Or is it a matter of aesthetics?

Reply:
It shouldn't have much effects downstream indeed, since the proof term is erased. The `erw` tactic tends to be a bit slower than `rw`, although it shouldn't be obviously so (after all, it's not doing much more than `exact` or `congr` already would).

My main motivation is maintainability/future-proofing: if for whatever reason the definition of `submonoid.comap` changes, then we'd have to fix up each application of `erw`. So I think it's better to explicitly state which identifications we are making during a proof. (Although I have to mention that I think defeq is bad for maintainability in general so I might be somewhat prejudiced against `erw`. On the other hand, defeq is what we get for using type theory :P)

Reply:
Thanks for the clarification. I get your point now.